core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -157,6 +159,12 @@ public StackV4Response post(@Valid @RequestObject DistroXV1Request request) {
         return distroxService.post(request);
```

Comment:
usually when we are creating an equivalent internal api, then one calls the other in controller to avoid double administration in the future